package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-test-with-data",
-  "version": "1.1.10",
+  "version": "1.2.0",
```

Comment:
Changed the version back to what it was. Just wanted to make sure that package.json has correct version.